src/shared/src/components/Select.tsx

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from '@material-ui/core/styles';
+import cn from 'classnames';
+import * as React from 'react';
+import { AltinnAppTheme } from '../theme';
+import '../../../altinn-app-frontend/src/styles/shared.css';
+
+const useStyles = makeStyles({
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+const optionStyle = {
+  display: 'none',
+};
+interface ISelectProps {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLSelectElement>) => void;
+  options: { value: string; label: string }[];
+  value: string;
+  disabled?: boolean;
+  error?: boolean;
+  fullWidth?: boolean;
+  className?: string;
+}
+
+const Select = ({
+  id,
+  onChange,
+  onBlur,
+  options,
+  value,
+  disabled = false,
+  error = false,
+  fullWidth = false,
+  className,
+}: ISelectProps) => {
+  const classes = useStyles();
+  return (
+      <select
+        id={id}
+        value={value}
+        disabled={disabled}
+        className={cn(
+          classes.select,
+          'custom-select a-custom-select',
```

Comment:
These classes seem to be only defined in the old design-system.
Maybe you could remove them and just add `width: '100%'` in the select class directly?

Reply:
This is how the select was implemented in DropDownComponent.tsx before i refactored the code into this Select component. If I remove these classes the dropdown looses its altinn look. 
![image](https://user-images.githubusercontent.com/43168787/169824744-38ee3e04-a225-4023-b04c-7b8de21a43b8.png)
to this: 
![image](https://user-images.githubusercontent.com/43168787/169824796-98331800-c66b-4b10-8f88-0593d2c527a9.png)